website/docs/r/schema_template_contract.html.markdown

```diff
@@ -14,16 +14,16 @@ Manages MSO Schema Template Contract.
 
 ```hcl
 resource "mso_schema_template_contract" "template_contract" {
```

Comment:
missing spaces (top and bottom) like you did on others

Reply:
ok